CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# CHANGELOG
+
+## Upcoming changes
+
+- feat: Develop Universal Robots (UR5e) Assets and RL Reach Environments (#1)
```

Comment:
I know it's a bit uncommon for how we maintain it in other places but it would be nice to be more specific here, namely that we changed the reach env cfg to include the ee frame.

Reply:
done!